ee-9/common/pom.xml

```diff
@@ -1536,6 +1536,18 @@
                 </exclusion>
             </exclusions>
         </dependency>
+
+        <dependency>
+            <groupId>${ee.maven.groupId}</groupId>
+            <artifactId>wildfly-webservices-opensaml-sysconfig</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>*</groupId>
+                    <artifactId>*</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
```

Comment:
This will need to be in ee-feature-pack/common/pom.xml as this module is back to only being used for 'preview' things.

Reply:
Fixed.